src/CodeGeneration.Roslyn.Tests.Generators/MultiplySuffixGenerator.cs

```diff
@@ -37,7 +37,7 @@ public Task<SyntaxList<MemberDeclarationSyntax>> GenerateAsync(TransformationCon
                         var propertySymbol = context.SemanticModel.GetDeclaredSymbol(x);
                         var attribute = propertySymbol?.GetAttributes()
                             .FirstOrDefault(a => a.AttributeClass.Name == nameof(TestAttribute));
-                        string suffix = "Suff" + string.Concat(attribute?.NamedArguments.Select(a => a.Value.Value.ToString()) ?? Enumerable.Empty<string>());
+                        string suffix = "Suff" + string.Concat(attribute?.NamedArguments.Select(a => a.Value.Value!.ToString()) ?? Enumerable.Empty<string>());
```

Comment:
Similar here: I don't think assuming the user didn't pass in null here is safe. Should we check for null and throw a more helpful exception message instead?

Reply:
Ah, but this is in tests, so I don't care.